e2e/cypress/integration/messaging/message_channel_draw_spec.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Draw plugin : Post message', () => {
+    before(() => {
+        //Login with admin access -> enable Draw plugin
+        cy.apiLogin('sysadmin').visit('/');
+        cy.switchToPluginManagementEnableDisableDraw('Enable', fileName, fileType);
+
+        // # Login, set display preference and go to /
+        cy.apiLogin('user-1').visit('/');
+        cy.apiSaveMessageDisplayPreference();
+    });
+
+    after(() => {
+        const newSettings = {
+
+            // # Restore default configuration
+            PluginSettings: {
+                Enable: true,
+                EnableMarketplace: true,
+                MarketplaceUrl: 'https://api.integrations.mattermost.com',
+            },
+        };
```

Comment:
Should it just be `Enable` to true? Others like `EnableMarketplace` and `MarketplaceUrl` might not be necessary.  Also, please do config update at `before` hook (not in `after`).

Reply:
This is not Required for this . Will remove